diff --git a/test.c b/test.c
index 441f670..5a0aa05 100644
--- a/test.c
+++ b/test.c
@@ -1,33 +1,28 @@
 // NOTE: Lines must be more than 30 characters long to activate copy/move
 // detection.

 if (100000000000000000000000000000)
 {
   // A
   // B
 }
-else if (200000000000000000000000000000)
+else if (500000000000000000000000000000)
 {
-  // C
-  // D
+  // I
+  // J
 }
 else if (300000000000000000000000000000)
 {
   // E
   // F
 }
 else if (400000000000000000000000000000)
 {
   // G
   // H
 }
-else if (500000000000000000000000000000)
-{
-  // I
-  // J
-}
 else if (600000000000000000000000000000)
 {
   // K
   // L
 }
